roles/openshift_logging/README.md

```diff
@@ -160,3 +161,18 @@ Elasticsearch OPS too, if using an OPS cluster:
   need to set this
 - `openshift_logging_mux_buffer_queue_limit`: Default `[1024]` - Buffer queue limit for Mux.
 - `openshift_logging_mux_buffer_size_limit`: Default `[1m]` - Buffer chunk limit for Mux.
+- `openshift_logging_mux_file_buffer_limit`: Default `[2Gi]` per output - Mux will
+  set the value to the file buffer limit.
```

Comment:
Not sure, but if "Mux will set the value to the file buffer limit," and this is the option for setting the mux file buffer limit, does this mean to say that it will calculate a default from the available diskspace considering the number of outputs?